pkg/apis/workspaces/v1alpha1/devworkspaceTemplateSpec.go

```diff
@@ -9,6 +9,10 @@ type DevWorkspaceTemplateSpec struct {
 	// Predefined, ready-to-use, workspace-related commands
 	Commands          []Command      `json:"commands,omitempty"`
 
+	// Bindings of commands to events
+	// +optional
```

Comment:
Are not commands and projects optional as well?

Reply:
Sure they are.
Just fixed it